core/arch/arm/kernel/thread_spmc.c

```diff
@@ -1873,16 +1902,17 @@ struct mobj_ffa *thread_spmc_populate_mobj_from_rx(uint64_t cookie)
 	struct mobj_ffa *mf = NULL;
 	unsigned int num_pages = 0;
 	unsigned int offs = 0;
+	unsigned int num_regions = 0;
```

Comment:
`range_count` would match the FF-A terminology better.

Reply:
Done